paddle/fluid/framework/tensor.cc

```diff
@@ -38,14 +42,20 @@ void* Tensor::mutable_data(const platform::Place& place,
                            proto::VarType::Type type, size_t requested_size) {
   type_ = type;
   PADDLE_ENFORCE_GE(numel(), 0,
-                    "When calling this method, the Tensor's numel must be "
-                    "equal or larger than zero. "
-                    "Please check Tensor::dims, or Tensor::Resize has been "
-                    "called first. The Tensor's shape is [",
-                    dims(), "] now");
+                    platform::errors::PreconditionNotMet(
+                        "The Tensor's numel must be "
+                        "equal or larger than zero. "
+                        "Please check Tensor::dims, or Tensor::Resize has been "
```

Comment:
user may not kown the `Tensor::dims` and `Tensor::Resize`, because they are interal method, we can only tell users the `The tensor's element number is illegal, it must be equal or larger than zero, but actually it's %d, and tensor shape is [%s]`

Reply:
done, thx